pages/collective-page.js

```diff
@@ -220,7 +222,7 @@ class CollectivePage extends React.Component {
                 />
               )}
             </CollectiveThemeProvider>
-            {LoggedInUser && mode === 'onboarding' && (
+            {isMember && mode === 'onboarding' && (
```

Comment:
Only an admin will be allowed to go through the onboarding, so we can directly write:

```suggestion
            {mode === 'onboarding' && LoggedInUser?.canEditCollective(collective) && (
```

Reply:
@Betree : Ah, nice catch. Thanks, I've corrected it. 😄